From: SJ Park <sj@kernel.org>
Date: Thu, 23 Jul 2026 06:56:09 -0700
Subject: [PATCH] ==== unurgent fixes ====

mm/damon: fix DAMOS bugs in core, paddr and vaddr

Fix misc bugs of DAMON.  Patch 1 make DAMOS less stress memory allocator
under extreme situation.  Patch 2 fixes wrong DAMOS quota auto-tuning
feedback loop start point for PSI goal.  Patches 3 and 4 fix wrong
folios walking in DAMON_PADDR.  Patches 5 and 6 fix wrong folios walking
in DAMON_VADDR.  Patches 7-9 handle extreme and unlikely memory
sitaution that can cause divide by zero and underflow.

All bugs are discovered by Sashiko.

The bugs are not very critical, but better to be merged sooner than
later.  I aim 7.3-rcX.  I will drop the RFC tag after 7.3-rc1.

Signed-off-by: SJ Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_txdd_30s | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_txdd_30s

diff --git a/hkml_cv_bogus/hkml_cv_bogus_txdd_30s b/hkml_cv_bogus/hkml_cv_bogus_txdd_30s
new file mode 100644
index 0000000000000..e69de29bb2d1d
-- 
2.47.3

